server/s3_object_metadata.cc

```diff
@@ -37,6 +37,18 @@
 
 extern struct m0_uint128 global_instance_id;
 
+std::shared_ptr<S3ObjectMetadata>
+S3ObjectMetadataFactory::create_delete_marker_metadata_obj(
+    std::shared_ptr<S3RequestObject> req,
+    const struct s3_motr_idx_layout& obj_idx_lo,
+    const struct s3_motr_idx_layout& obj_ver_idx_lo) {
+
+  std::shared_ptr<S3ObjectMetadata> meta =
```

Comment:
```suggestion
  auto meta =
```

`auto` is useful when the function makes it obvious what the type is.

Reply:
Updated